package.json

```diff
@@ -833,6 +833,11 @@
                             ""
                         ]
                     },
+                    "azureFunctions.projectLanguageModel": {
+                        "scope": "resource",
+                        "type": "number",
```

Comment:
Do we need any constraints on the values allowed? e.g. `maximum` or `minimum`?

Reply:
There's no theoretical maximum, but I'll set a `minimum` of 1.